packages/gtk3.rb

```diff
@@ -3,25 +3,23 @@
 class Gtk3 < Package
   description 'GTK+ is a multi-platform toolkit for creating graphical user interfaces.'
   homepage 'https://developer.gnome.org/gtk3/3.0/'
-  @_ver = '3.24.34-80d8970'
+  @_ver = '3.24.37'
   @_ver_prelastdot = @_ver.rpartition('.')[0]
```

Comment:
Why do we need this?  It's not referenced anywhere in the package.

Reply:
It was once... I can get rid of it...